examples/custom-ipld-formats/daemon-node.js

```diff
@@ -5,7 +5,7 @@
 const codecName = 'dag-test'
 const codecNumber = 392091
 
-const baseTable = require('multicodec/src/base-table.json')
+const { baseTable } = require('multicodec/src/base-table')
```

Comment:
Not sure this is working as expected.  CI says:

```
example-custom-ipld-formats: Error: `put` requires a format
example-custom-ipld-formats:     at IPLDResolver.put (/home/travis/build/ipfs/js-ipfs/node_modules/ipld/src/index.js:180:13)
example-custom-ipld-formats:     at put (/home/travis/build/ipfs/js-ipfs/node_modules/ipfs-core/src/components/dag/put.js:75:30)
example-custom-ipld-formats:     at Object.put (/home/travis/build/ipfs/js-ipfs/node_modules/ipfs-core/src/utils.js:231:46)
example-custom-ipld-formats:     at main (/home/travis/build/ipfs/js-ipfs/examples/custom-ipld-formats/in-process-node.js:49:30)
```

Reply:
I can try to see what is happening here in a couple of hours

Reply:
So, an Object.freeze is exported from base-table. This means that the object is not changed by the following change